doc/mgr/prometheus.rst

```diff
@@ -184,6 +185,16 @@ Example to turn up the sync interval to 10 minutes::
 
   ceph config set mgr mgr/prometheus/rbd_stats_pools_refresh_interval 600
 
+Ceph daemons performance counters metrics
+-----------------------------------------
+
+With the introduction of ceph-exporter daemon, the module will no longer export ceph daemons
```

Comment:
introduction of the \`\`ceph-exporter\`\` daemon

no longer export Ceph daemon perf counters as Prometheus

However, one may re-enable exporting these metrics by setting the module option \`\`exclude_perf_counters\`\` to \`\`false\`\`

Reply:
This changes the meaning, because with the introcution of the ceph-exporter daemon, prometheus module no longer...